packaging/docker/Dockerfile.ubuntu

```diff
@@ -0,0 +1,56 @@
+ARG BASE_IMAGE=ubuntu:18.10
+FROM ${BASE_IMAGE} AS grafana-builder
+
+ARG GRAFANA_TGZ="grafana-latest.linux-x64.tar.gz"
+
+COPY ${GRAFANA_TGZ} /tmp/grafana.tar.gz
+
+RUN mkdir /tmp/grafana && tar xfz /tmp/grafana.tar.gz --strip-components=1 -C /tmp/grafana
+
+FROM ${BASE_IMAGE}
+
+EXPOSE 3000
+
+ARG GF_UID="472"
+ARG GF_GID="472"
```

Comment:
The 6.3.x one have `ARG DEBIAN_FRONTEND=noninteractive` - don't we need that?

Reply:
@marefr Shouldn't that be ENV instead of ARG? It sounds as if you want to set the environment variable `DEBIAN_FRONTEND`, in which case `ENV` would be right. I can add the ENV as I guess it can't hurt, even though we don't notice any need for it right now.

Reply:
I have no idea actually. Just spotted that it was in the old dockerfile

Reply:
@marefr I added a corresponding `ENV` directive. It doesn't seem to matter now, but might help in the future.

Reply:
Did some further investigation and sounds like we don't want to add this: https://github.com/moby/moby/issues/4032#issuecomment-34597177

Reply:
@marefr Looks as if `ARG` actually sets the environment variable during build?? https://github.com/moby/moby/issues/4032#issuecomment-192327844

Had never heard of this until now, kind of magical since ARG primarily is for taking user variables.

Reply:
ARG specifies an environment variable for the build commands, and may be overridden from the command line during docker build.  ENV specifies an environment variable that will be set inside the container itself when it runs (and will be available to the build commands as well).  This is a good primer: https://vsupalov.com/docker-arg-env-variable-guide/

Setting it via ARG is a good idea, setting it via ENV is not.